WordPress/Sniffs/WP/EnqueuedResourcesSniff.php

```diff
@@ -1,65 +0,0 @@
-<?php
```

Comment:
I'm not sure you intended to remove this sniff. If you did, could you please explain why? I think that it needs to stay, because it checks to make sure that the `wp_enqueue_*()` functions are actually used, instead of hard-coded `script`/`style` tags.

Reply:
Based on the presence of a seemingly duplicate sniff, and the seemingly accidental removal of this one, I think it was a simple mistake of deleting the wrong file.

Reply:
Good point; that's easy to do. 😄